workshops/templates/workshops/person_bulk_add_results.html

```diff
@@ -51,7 +51,6 @@
                 {% endfor %}
                 </ul>
             {% else %}
-                {{ entry.errors|default_if_none:"&mdash;" }}
```

Comment:
I kind of like the dash, but there's no clear motivation for it when it landed in 9e44195 (Bulk-edit individual fields on confirmation page, 2015-01-14), so I guess it's ok to remove.  If it goes, you should remove the `{% else %}` too.